src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +35,41 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={() => {
+                selectTodo({
+                  id: 0,
+                  title: '',
+                  completed: false,
+                  userId: 0,
+                });
+              }}
             />
           </header>
 
           <div className="modal-card-body">
             <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
+              {title}
             </p>
 
             <p className="block" data-cy="modal-user">
-              {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
+              {completed ? (
+                <strong className="has-text-success">Done</strong>
+              ) : (
+                <strong className="has-text-danger">Planned</strong>
+              )}
 
               {' by '}
 
-              <a href="mailto:Sincere@april.biz">
-                Leanne Graham
+              <a href={`mailto:${user.email}`}>
```

Comment:
use destructuring

Reply:
I can't use destructuring 
![image](https://user-images.githubusercontent.com/107570670/196415142-3efb0314-03ae-482b-8302-1b01ad87cf76.png)

Reply:
The default state user is null. Before I get a data about a user from the server (after first render) I get error all time. Why I need to use restructuring of the state? 
![image](https://user-images.githubusercontent.com/107570670/196962186-95715773-f870-47cf-92bb-b4621bc1dbc2.png)
![image](https://user-images.githubusercontent.com/107570670/196962282-685f5837-dc9a-4e96-b7ee-aca78daca491.png)